allvers/2005-05-11-GPL-2.0-from-fsf.org.txt.notes

```diff
@@ -0,0 +1,3 @@
+Link: http://web.archive.org/web/20050511030123/http://www.fsf.org:80/licensing/licenses/gpl.txt
+Date: 20050511
+Notes:  "Library" to "Lesser" in appendix only  only on fsf.org
```

Comment:
Incorrect. This one does not have "Lesser", that's a later version.

Reply:
Corrected in 6c1f322be27f70321bc974e693276f41db5c052e